extensions/Squidex.Extensions/Actions/Email/EmailActionHandler.cs

```diff
@@ -0,0 +1,74 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Squidex.Domain.Apps.Core.HandleRules;
+using Squidex.Domain.Apps.Core.HandleRules.EnrichedEvents;
+using Squidex.Infrastructure;
+
+namespace Squidex.Extensions.Actions.Email
+{
+    public sealed class EmailActionHandler : RuleActionHandler<EmailAction, EmailJob>
+    {
+        private const string Description = "Send an Email";
+
+        private readonly EmailOptions emailOptions;
+
+        public EmailActionHandler(RuleEventFormatter formatter, IOptions<EmailOptions> emailOptions)
```

Comment:
I would add the email options to the action, because it cannot be used with the cloud otherwise.

Reply:
Yes, good idea. i will move those options to action, thanks